homeassistant/components/ozw/services.py

```diff
@@ -37,6 +44,103 @@ def async_register(self):
             ),
         )
 
+        self._hass.services.async_register(
+            LOCK_DOMAIN,
```

Comment:
You're not allowed to register services under other domains. This needs to be OZW domain. But since it's all locks, just do it in the lock platform 👍

Reply:
Resolved in latest commit.